Mathlib/SetTheory/Game/PGame.lean

```diff
@@ -985,6 +994,42 @@ theorem equiv_congr_right {x₁ x₂ : PGame} : (x₁ ≈ x₂) ↔ ∀ y₁, (x
   ⟨fun h _ => ⟨fun h' => Equiv.trans (Equiv.symm h) h', fun h' => Equiv.trans h h'⟩,
    fun h => (h x₂).2 <| equiv_rfl⟩
 
+theorem Equiv.ext {x y : PGame}
```

Comment:
Assuming these `Equiv` lemmas are new, they seem like yet another prime candidate for splitting into their own PR. Note that one of my own PRs #21086 aims to deprecate `Equiv`, so these should be renamed in a way that doesn't use dot notation. (They can still use `equiv` for `≈`, though).